Formula/envoy.rb

```diff
@@ -29,7 +29,7 @@ def install
       --show_task_finish
       --verbose_failures
       --action_env=PATH=/usr/local/bin:/opt/local/bin:/usr/bin:/bin
```

Comment:
```suggestion
      --action_env=PATH=#{HOMEBREW_PREFIX}/bin:/usr/bin:/bin
```
If we're making further changes here we should probably do this, along with making the `%w` into a `%W` to allow interpolation. This might actually help this get bottled on Linux; maybe.

Reply:
cool